src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
```

Comment:
remove static

Reply:
After removing static from Ball getRandomBall() method the program does not want to compile and show (non-static method getRandomBall() cannot be referenced from a static context) in Main method (Ball ball1 = Lottery.getRandomBall(); )